src/components/maxwolts/style.css

```
@@ -0,0 +1,313 @@
+.maxwolts-panda-pass-main {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	height: 100vh;
+	width: 100vw;
+	background-color: white;
+	font-family: Verdana, Geneva, Tahoma, sans-serif;
+	font-size: 22px;
+}
+
+.maxwolts-panda-pass {
+	position: relative;
+	width: 100vw;
+	min-width: 300px;
+	max-width: 500px;
+	margin: auto;
+	background-color: #f9f9f9;
+	border: 1px solid #161616;
+	border-radius: 2%;
+	overflow: hidden;
+}
+
+.maxwolts-panda-pass-section-selector button {
+	width: 50%;
+	font-size: 22px;
+	padding: 10px;
+	transition: background 0.3s;
+}
+
+.maxwolts-selected-section {
+	background-color: #f9f9f9;
+	color: #161616;
+}
+
+.maxwolts-unselected-section {
+	background-color: #161616;
+	color: #f9f9f9;
+}
+
+.maxwolts-input-range {
+	padding: 32px 10%;
+}
+
+.maxwolts-result {
+	position: relative;
+	overflow: hidden;
+	margin: 0 10%;
+	border: 1px solid #161616;
+	perspective: 1000px;
+}
+
+.maxwolts-result-content {
+	position: relative;
+	display: grid;
+	grid-template-columns: 1fr 44px;
+	padding: 10px;
+	background-color: #f9f9f9;
+	transform-style: preserve-3d;
+	transform-origin: bottom;
+	transition: transform 1s;
+}
+
+.maxwolts-result-content input {
+	outline: none;
+	overflow: hidden;
+}
+
+.maxwolts-animation-panda-result {
+	animation: animation-rotate 2s ease-out;
+}
+
+@keyframes animation-rotate {
```

Holaa!! Muchas gracias por cambiarlos!! 🚀 Quedó un keyframe solitario sin el scope 👀 Solo faltaría eso y ya podremos mergearlo! 💪🏻
```suggestion
@keyframes maxwolts-animation-rotate {
```

---

Hi!, listo el cambio fue hecho. 😃